app/models/factory_log.rb

```diff
@@ -18,15 +19,16 @@ def self.ranking
     results = []
     FactoryLog.all.each do |log|
       time = log.execution_time
-      factory = log.factory
-      if index = results.index { |cd| cd[:factory] == factory }
+      #毎回これやってるの不要では？メモ化できるのでは
+      #というかFactoryと連結する必要あんまりないのか。
+      if index = results.index { |cd| cd[:factory] == log.factory_name }
         hash_data = results[index]
         hash_data[:total_time] += time
         hash_data[:count] += 1
         hash_data[:average_time] = hash_data[:total_time] / hash_data[:count]
       else
         results.push(
-                      factory: factory,
+                      factory: log.factory_name,
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the previous line.